package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@lightninglabs/lnc-core",
-  "version": "0.1.12-alpha",
+  "version": "0.2.0-alpha",
   "description": "Type definitions and utilities for Lightning Node Connect",
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "config": {
-    "lnd_release_tag": "v0.15.1-beta",
-    "loop_release_tag": "v0.20.1-beta",
-    "pool_release_tag": "v0.5.8-alpha",
+    "lnd_release_tag": "v0.15.4-beta",
+    "loop_release_tag": "v0.20.2-beta",
+    "pool_release_tag": "v0.6.1-beta",
```

Comment:
When updating these versions, I don't think we can always jump to the latest available. We have to make sure that both `litd` and `lightning-node-connect` have also been updated first. If the protos here don't match the other two repos, then you'll get a marshalling error when calling RPCs that differ.

It looks like [litd](https://github.com/lightninglabs/lightning-terminal/blob/master/go.mod) and [LNC](https://github.com/lightninglabs/lightning-node-connect/blob/master/cmd/wasm-client/go.mod) are still using Pool v0.5.8-alpha and Loop v0.20.1-beta, so I think we should use those versions here as well.

Reply:
Good catch. Will remember this moving forward.